### DISCUSSION TITLE
Welcome to DSPFFT Discussions!

### DISCUSSION THREAD

**Discussion by Timothy-Liuxf** [Timothy-LiuXuefeng/DSPFFT#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yokeman-young** [comment]
不知道性能之间测试如何
https://github.com/yangyongkang2000/yyk_fft
